src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -208,7 +207,14 @@ struct WgVertexBuffer
                 pntIndex++;
             } else if (cmd == PathCommand::Close) {
                 close();
+                // close command creates the new object, started form the closing point
+                if (update_dist) updateDistances();
```

Comment:
but it makes sense only if there is a next point and it is not a 'move to', right? maybe check for that?

Reply:
fixed